ush/parsing_model_configure_FV3.sh

```diff
@@ -41,12 +40,11 @@ restart_interval:        $restart_interval
 quilting:                $QUILTING
 write_groups:            ${WRITE_GROUP:-1}
 write_tasks_per_group:   ${WRTTASK_PER_GROUP:-24}
-output_history:          ${OUTPUT_HISTORY:-".true."}
```

Comment:
Should output_history and write_dopost be left in too?

Reply:
Okay, I see this below now in the "if clp" section.   the atm_coupling_interval_sec is something needed for coupling, but I thought the output_history and write_dopost did not have anything to do with coupling, so I would think those should be left in the main section or removed.